test/parallel/test-crypto-random.js

```diff
@@ -37,6 +37,15 @@ process.setMaxListeners(256);
   });
 });
 
+// Test to make sure addSystemEntropy returns in less than 10 ms.
+(function() {
+  var start = process.hrtime();
+  crypto.addSystemEntropy();
+  var duration = process.hrtime(start);
+  duration = duration[0] * 1e9 + duration[1]; // convert to nanoseconds
+  assert.ok(duration < 10 * 1e6, 'addSystemEntropy responds in < 10ms');
```

Comment:
I speculate that this is going to be flaky on the slower buildbots.  Can you compare against `common.platformTimeout(10) * 1e6`?